wamr-compiler/main.c

```diff
@@ -266,6 +268,9 @@ main(int argc, char *argv[])
         else if (!strcmp(argv[0], "--enable-indirect-mode")) {
             option.is_indirect_mode = true;
         }
+        else if (!strcmp(argv[0], "--enable-gc")) {
+            option.enable_gc = true;
```

Comment:
Should disable ref_types when gc is enabled?
```C
option.enable_ref_types = false;
```

And when handling `else if (!strcmp(argv[0], "--enable-ref-types")) {`
```C
option.enable_ref_types = true;
option.enable_gc = false;
```

Reply:
OK, this case handled in line 333 by:
```c
    if (option.enable_gc) {
        option.enable_ref_types = false;
    }
```